pkg/service/generate/generate.go

```diff
@@ -0,0 +1,83 @@
+package generate
+
+import (
+	"sync"
+	"os"
+	"path/filepath"
+
+	"go.keploy.io/server/pkg/models"
+	
+	"gopkg.in/yaml.v3"
+	"go.uber.org/zap"
+)
+
+var Emoji = "\U0001F430" + " Keploy:"
+
+type generator struct {
+	logger *zap.Logger
+	mutex  sync.Mutex
+}
+
+func NewGenerator(logger *zap.Logger) Generator {
+	return &generator{
+		logger: logger,
+		mutex:  sync.Mutex{},
+	}
+}
+
+var globalNoise = `
+{
+	"global": {
+		"body": {},
```

Comment:
Is it possible to introduced comments here. User need to know which is mandatory field, like appCmd is mandatory. If we can mentioned a comment "mandatory field" above the field. That would be good.

Reply:
done! 
any other place where comments could be useful ?